includes/api/class-wc-admin-rest-product-variations-controller.php

```diff
@@ -22,4 +22,69 @@ class WC_Admin_REST_Product_Variations_Controller extends WC_REST_Product_Variat
 	 * @var string
 	 */
 	protected $namespace = 'wc/v4';
+
+	/**
+	 * Get the query params for collections.
+	 *
+	 * @return array
+	 */
+	public function get_collection_params() {
+		$params                   = parent::get_collection_params();
+		$params['product_search'] = array(
```

Comment:
Same question as https://github.com/woocommerce/woocommerce-admin/pull/1791/files#r265597768:

Do we have to use `product_search` instead of `search` here? Since we're already overriding `get_collection_params()`, can't we handle the `search` parameter there?

Reply:
@jeffstieler I was wondering the same.  My concern was that other plugins taking advantage of the `search` param might run into unexpected behavior, but maybe this is okay given that we've namespaced the API to v4.

Reply:
Yeah I think the namespacing saves us here - we're changing things much more drastically in the customers endpoint: https://github.com/woocommerce/woocommerce-admin/blob/master/includes/api/class-wc-admin-rest-customers-controller.php#L18